src/components/Menus/Sidebar/ProjectsPanel/ProjectsPanel.scss

```
@@ -21,10 +21,6 @@
     }
   }
 
-  .projects-panel__button {
-    padding: $space-1 0;
-  }
-
   .projects-panel__download-button {
     width: 100%;
     margin: 0;
```

worth just centring the content to align a little closer to design 👍 

```suggestion
    margin: 0;
    justify-content: center;
```